solutions/devsprint-michel-ferreira-2/src/main/kotlin/framework/ResourceReader.kt

```diff
@@ -0,0 +1,17 @@
+package framework
+
+import framework.interfaces.IResourceReader
+
+object ResourceReader : IResourceReader, Loggable() {
+
+    override fun readAsList(fileName: String): List<String> {
+        return try {
+            val message = "File ${fileName.replace("/", "")} not found!"
+            val resource = this::class.java.getResource(fileName) ?: throw Exception(message)
```

Comment:
boa!
Depois podemos criar uma Exception específica para tratar not found

Reply:
Se já quiser aproveitar o commit para criar essa exception, eu já fiz o merge da OwnedException na nossa branch dev